src/cript/api/api.py

```diff
@@ -135,6 +164,17 @@ def __init__(self, host: Union[str, None], token: [str, None]):
             Instantiate a new CRIPT API object
         """
 
+        if config_file_path and not host and not token:
```

Comment:
Maybe this should be a little more elaborate.
I.e. if a user specifies host via an argument (or env variable) but the `token` via the config file, this should be OK.

If you are asked me, config file should take 3. prioty.

1. direct arguments.
2. if direct arguments, try env variable
3. if direct argument and env variable not available take config file.

Reply:
I think that list makes sense. Mine was config file as priority and I can swing either way for direct and env vars. I'm thinking because config file is easy to setup and encourages user to take out arguments out of their code so they can easily share it.

what are your thoughts, I can swing this either way, I don't have a strong preference one way or another

Reply:
I think for v1, I'm not going to allow for reading the host from config and token from environment variable, either put all into environment variable, config, or directly and in next versions we can start allowing for more flexbility